CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+1.9.0 / 2020-04-14
+========================
+* ARM build for velero-plugin, ARM images is published under openebs/velero-plugin-arm64 ([#61](https://github.com/openebs/velero-plugin/pull/61),[@akhilerm](https://github.com/akhilerm))
```

Comment:
grammar error : `ARM image is` or `ARM images are`

Reply:
done